pandas/io/tests/test_sql.py

```diff
@@ -236,7 +236,7 @@ def _close_conn(self):
         pass
 
 
-class PandasSQLTest(unittest.TestCase):
+class PandasSQLTest(object):
```

Comment:
Thanks. This is the base class, there aren't any `test_` methods attached to `PandasSQLTest`. All of the real test classes, which inherit from `PandasSQLTest` *do* start with with `Test`. IIRC the root problem was the setUp methods weren't being run with the prior inheritance. I'll post a fuller explanation later, because the diff isn't very informative.